tests/units/offline/test_mobile_base.py

```diff
@@ -0,0 +1,87 @@
+import grpc
+import pytest
+from google.protobuf.wrappers_pb2 import BoolValue, FloatValue
+from reachy2_sdk_api.mobile_base_lidar_pb2 import (
+    LidarObstacleDetectionEnum,
+    LidarObstacleDetectionStatus,
+    LidarSafety,
+)
+from reachy2_sdk_api.mobile_base_utility_pb2 import (
+    BatteryLevel,
+    ControlModeCommand,
+    ControlModePossiblities,
+)
+from reachy2_sdk_api.mobile_base_utility_pb2 import MobileBase as MobileBase_proto
+from reachy2_sdk_api.mobile_base_utility_pb2 import (
+    MobileBaseInfo,
+    MobileBaseState,
+    ZuuuModeCommand,
+    ZuuuModePossiblities,
+)
+
+from reachy2_sdk.parts.mobile_base import MobileBase
+
+
+@pytest.mark.offline
+def test_class() -> None:
```

Comment:
Same here.
__repr__
_set_drive_mode,  _set_control_mode, set_speed, goto, goto_async : cannot test good values because it requires a stub, but you can test bad ones to see if the try catch works
turn_on, turn_off